src/pages/invite/InviteBot.tsx

```diff
@@ -38,71 +73,113 @@ export default function InviteBot() {
                             <h1>{data.username}</h1>
-                            {data.description && (
-                                <Markdown content={data.description} />
-                            )}
                         </div>
+                        {data.description && (
+                            <div className="description">
+                                <Markdown content={data.description} />
+                            </div>
+                        )}
                     </BotInfo>
-                    <Category>Add to server</Category>
                     <Option>
-                        <ComboBox
-                            value={server}
-                            onChange={(e) => setServer(e.currentTarget.value)}>
-                            <option value="none">Select a server</option>
-                            {[...client.servers.values()]
-                                .filter((x) => x.havePermission("ManageServer"))
-                                .map((server) => (
-                                    <option value={server._id} key={server._id}>
-                                        {server.name}
-                                    </option>
-                                ))}
-                        </ComboBox>
-                        <Button
-                            palette="secondary"
-                            onClick={() =>
-                                server !== "none" &&
-                                client.bots.invite(data._id, { server })
-                            }>
-                            Add
-                        </Button>
-                    </Option>
-                    <Category>Add to group</Category>
-                    <Option>
-                        <ComboBox
-                            value={group}
-                            onChange={(e) => setGroup(e.currentTarget.value)}>
-                            <option value="none">Select a group</option>
-                            {[...client.channels.values()]
-                                .filter((x) => x.channel_type === "Group")
-                                .map((channel) => (
-                                    <option
-                                        value={channel._id}
-                                        key={channel._id}>
-                                        {channel.name}
-                                    </option>
-                                ))}
-                        </ComboBox>
-                        <Button
-                            palette="secondary"
-                            onClick={() =>
-                                group !== "none" &&
-                                client.bots.invite(data._id, { group })
-                            }>
-                            Add
-                        </Button>
+                        <Category>
+                            Add this bot to your Revolt server, group
```

Comment:
Reword -> `Add this bot to your server or group`

Reply:
oh yeah that look better